apps/fluent-tester/src/FluentTester/TestComponents/ActivityIndicator/ActivityIndicatorTest.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { ActivityIndicator } from '@fluentui-react-native/experimental-activity-indicator';
+import { Text } from '@fluentui/react-native';
+import { Stack } from '@fluentui-react-native/stack';
+import { stackStyle } from '../Common/styles';
+import { Test, TestSection, PlatformStatus } from '../Test';
+import { ACTIVITYINDICATOR_TESTPAGE } from './consts';
+
+const activityIndicatorTest: React.FunctionComponent<{}> = () => {
+  /** Bug: Customize doesn't do anything
+   * Tried having tokens not be props, but didn't work
+   * Not sure how to test/check where the tokens are passed through other than just looking at the final render
+   */
+  const CustomizedActivityIndicator = ActivityIndicator.customize({
+    activityIndicatorColor: '#458792',
```

Comment:
Is there a more descriptive name for this hex value? We (Apex generally prefer to put names for hardcoded values for clarity.

Reply:
I just put a random color that would stand out, but I can change it to a named color instead.